content/guides/references/migration-guide.md

```diff
@@ -657,70 +669,46 @@ Here are a few examples, assuming the value of `videosFolder` is
   - `cypress/videos/file/one.cy.js.mp4`
   - `cypress/videos/two.cy.js.mp4`
 
-### Command / Cypress API changes
+### Command / Cypress API Changes
 
 #### `cy.mount()`
 
-A default `cy.mount()` command has been added. Note that this command only warns
-and redirect users to the API page, the user still needs to manually define
-their own `cy.mount()` command with `Cypress.Commands.add('mount', ...)`.
+A default `cy.mount()` command has been added for mounting components in
+Component Testing. Note that this command only warns and redirect users to the
+API page, the user still needs to manually define their own `cy.mount()` command
```

Comment:
can you explain why we added a default command that doesn't do anything?Reading this without context make this seem like a super weird decision on our part.

Reply:
It does do something, in that it makes `cy.mount` globally available to use in CT tests. Our suggestion and references all throughout the docs use `cy.mount`, so if it wasn't defined, the user would get an error when attempting to call it.

Reply:
Oh I see what you are saying. It was modeled after the `hover` command, where instead of erroring saying that the command isn't available, we say you have to set it up manually and provide a link to the docs on how to do so.